generators/client/files-angular.js

```diff
@@ -111,6 +110,13 @@ const files = {
             templates: [
                 'blocks/interceptor/_auth.interceptor.ts'
             ]
+        },
+        {
+            condition: generator => !generator.skipServer,
```

Comment:
Actually I wouldnt worry much about this as if you are skipping server then you need to make sure you are providing endpoints for all the REST apis else everything will fail so its not really important to validate this. but since you have done it its ok